### PR TITLE
Makefile has been updated

### PR DESCRIPTION
**Pull request by jacobsakran**
The command "make help" now shows a help screen for all the commands.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -10,63 +10,71 @@ ES_IMAGE_TAG=docker.io/library/elasticsearch:6.8.12
...
 	docker push $(IMAGE_PREFIX)/cluster-logging-load-client:$(IMAGE_TAG)
 	docker push $(IMAGE_PREFIX)/cluster-logging-load-client:latest
 
-deploy:
+deploy: ## Deploy the image (build-image must be called before deploy)
 	kubectl apply -f deployment.yaml
 
-undeploy:
+help: ## Display this help
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-20s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
```
Comment (eranra) [inline Makefile:44]: change justify spaces to 25 so that the lines will be aligned when using `make help`

File: Makefile [hunk 2]
```diff
@@ -10,63 +10,71 @@ ES_IMAGE_TAG=docker.io/library/elasticsearch:6.8.12
...
 
-clean-local:
+clean-local: ## Clean all the local containers
 	podman kill $(ES_CONTAINER_NAME) > /dev/null 2>&1 || true
 	podman rm $(ES_CONTAINER_NAME) > /dev/null 2>&1 || true
 	podman kill $(LOKI_CONTAINER_NAME) > /dev/null 2>&1 || true
 	podman rm $(LOKI_CONTAINER_NAME) > /dev/null 2>&1 || true
 
-run-local-es-generate:
+run-local-es-generate: ## Run logger and with remote type elasticsearch
```
Comment (eranra) [inline Makefile:58]: drop the word "and" from the remark

File: Makefile [hunk 3]
```diff
@@ -10,63 +10,71 @@ ES_IMAGE_TAG=docker.io/library/elasticsearch:6.8.12
...
 	./logger generate --log-level info --destination elasticsearch --destination-url http://localhost:9200/ --totalLogLines 5
 
-run-local-es-query:
+run-local-es-query: ## Generate query requests to elasticsearch v6
 	./logger query --log-level info --destination elasticsearch --destination-url http://localhost:9200/ --query-file ./config/es_queries.yaml --totalLogLines 2
 
-run-local-loki-generate:
+run-local-loki-generate: ## Run logger and set with remote type loki
```
Comment (eranra) [inline Makefile:64]: drop the word "and" from the remark

File: Makefile [hunk 4]
```diff
@@ -10,63 +10,71 @@ ES_IMAGE_TAG=docker.io/library/elasticsearch:6.8.12
...
 
-undeploy:
+help: ## Display this help
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-20s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+undeploy: ## Undeploy the image
 	kubectl delete -f deployment.yaml
 
-local: clean-local deploy-local-es deploy-local-loki run-local-es-generate run-local-es-query run-local-loki-generate run-local-loki-query
+local: ## Run all the local commands
```
Comment (eranra) [inline Makefile:49]: You can add in the remark that this is a "local usage example" of logger

Comment (eranra) [inline Makefile:49]: you need to move all dependent targets to the same line :-( ... please fix in all places 

local:  clean-local deploy-local-es deploy-local-loki run-local-es-generate run-local-es-query run-local-loki-generate run-local-loki-query ## Run all the local commands

File: Makefile [hunk 5]
```diff
@@ -10,63 +10,71 @@ ES_IMAGE_TAG=docker.io/library/elasticsearch:6.8.12
...
 	 $(GOLINT) main.go
 
-build: lint
+build: ## Build the executable
+	make lint
 	go build -o logger -v main.go
 
-test: lint
+test: ## Run the tests
+	make lint
```
[the remark this answers is not in the document]
Comment (jacobsakran) [reply Makefile:25]: Done

File: Makefile (hunk 2 above)
Comment (jacobsakran) [reply Makefile:58]: Done

File: Makefile (hunk 3 above)
Comment (jacobsakran) [reply Makefile:64]: Done

Comment (jacobsakran) [reply Makefile:64]: Took this comment from: https://github.com/ViaQ/cluster-logging-load-client
It's written in the README, in the section ##Loki
(I think that should be edited too)

File: Makefile (hunk 1 above)
Comment (jacobsakran) [reply Makefile]: Done

**Pull request closed without merging**